docs/src/design/stack/field_ops.md

```diff
@@ -178,19 +178,25 @@ The effect on the rest of the stack is:
 * **No change** starting from position $1$.
 
 ## EXPACC
-The `EXPACC` operation pops top $4$ elements from the top of the stack, performs a single round of exponent aggregation, and pushes the resulting $4$ values onto the stack. The diagram below illustrates this graphically.
+The `EXPACC` operation computes one round of the expression $base^{exp}$. It is expected that `Expacc` is called at least `num_exp_bits` times, where `num_exp_bits` is the number of bits required to represent `exp`.
+
+It pops top $4$ elements from the top of the stack, performs a single round of exponent aggregation, and pushes the resulting $4$ values onto the stack. The diagram below illustrates this graphically.
 
 ![expacc](../../assets/design/stack/field_operations/EXPACC.png)
 
+Expacc is based on the observation that the exponentiation of a number can be computed by repeatedly squaring the base and multiplying the accumulator by the base when the least significant bit of the exponent is 1.
```

Comment:
nit: the usage of base to signify the original base and the accumulated base holding the intermediate results is a bit confusing

Reply:
Good point. Renamed `base` -> `base_acc`, and `acc` -> `result_acc` (to disambiguate with the new `base_acc`).

Even with this change though, I find the English version of the algorithm written here still hard to read, so hopefully the example is enough to make things more clear.